### PR TITLE
[MDS-4968] Incidents text updates

### PR DESCRIPTION
**Pull request by matbusby-fw**
## Objective 

[MDS-4968](https://bcmines.atlassian.net/browse/MDS-4968)

Updated incidents text in as per the following instructions

Incidents overview table:

Change “Click here for more information on mining incidents and dangerous occurrences in British Columbia” to “Click here for more information on reportable incidents”

▶ Record a mine incident ▶ Get started

Paragraphs 1 and 3: Change both “(Mines code, section 1.7.1(1)(a))” and “(Mines code, section 1.7.1(1)(b))” to “(HSRC 1.7.2)”

Paragraphs 5 and 7: Change “Mines code” to “HSRC”

▶ Review Details → Heading: Record Files (Final report area)

Change” You determined that this incident was a dangerous occurrence. Please add your final report documentation by clicking below” to “Pursuant to section 1.7.2 of the HSRC, an investigation report must be submitted within 60 days of the reportable incident. Please add the final report documentation by clicking below”

### CODE REVIEW

File: services/minespace-web/src/components/dashboard/mine/incidents/Incidents.js [hunk 1]
```diff
@@ -134,7 +134,7 @@ const Incidents = (props) => {
           >
             Click here
           </a>
-          for more information on mining incidents and dangerous occurrences in British Columbia.
+          for more information on reportable incidents.
```
Comment (taraepp) [inline services/minespace-web/src/components/dashboard/mine/incidents/Incidents.js:137]: It does bother me that IncidentForm has "Please add the final report documentation by clicking below" (no period at end) and this one has "Click here for more information on reportable incidents." (period at end of statement). Do we know for sure which style is preferred?

Comment (matbusby-fw) [reply services/minespace-web/src/components/dashboard/mine/incidents/Incidents.js:137]: Hmmm, yes, that is inconsistent.  I'm not sure if there is a preferred style here.  I think removing the periods would be the way to go
image

File: services/minespace-web/src/components/pages/Incidents/IncidentGetStarted.js [hunk 2]
```diff
@@ -11,26 +11,42 @@ export const IncidentGetStarted = () => {
...
-          Call the <b>Mine Incident Reporting Line at </b>
+          Call the 
+          {' '}
+          <b>Mine Incident Reporting Line at </b>
           <a href="tel:1-888-348-0299">
             <b>1-888-348-0299</b>
-          </a>{" "}
+          </a>
+          {" "}
           . This line is monitored 24/7 by an On-call Inspector.
```
Comment (taraepp) [inline services/minespace-web/src/components/pages/Incidents/IncidentGetStarted.js:36]: this looks like there's a space before the period, is that how it renders?

File: services/core-web/src/components/Forms/incidents/IncidentForm.js [hunk 3]
```diff
@@ -508,8 +508,9 @@ const renderDocumentation = (childProps, isEditMode, handlers, parentHandlers) =
                   This incident requires a final investigation report.
                 </Typography.Paragraph>
                 <Typography.Paragraph>
-                  You determined that this incident was a dangerous occurence. Please add your final
-                  report documentation by clicking below.
+                  Pursuant to section 1.7.2 of the HSRC, an investigation report must be submitted
+                  within 60 days of the reportable incident. Please add the final report
+                  documentation by clicking below
```
Comment (taraepp) [inline services/core-web/src/components/Forms/incidents/IncidentForm.js]: can we add a period after "below"

File: services/minespace-web/src/components/pages/Incidents/IncidentGetStarted.js (hunk 2 above)
Comment (matbusby-fw) [reply services/minespace-web/src/components/pages/Incidents/IncidentGetStarted.js:36]: Ack, damn IDE

**Review by taraepp — APPROVED** [review]
👍

**Pull request merged**